src/makeRobot.js

```diff
@@ -38,7 +38,74 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const obj = {
+    name: name,
+    wheels: wheels,
+    version: version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
```

Comment:
```suggestion
    name,
    wheels,
    version,
    coords: {
      x: 0,
      y: 0,
    },
```
If the property key has the same name as a variable, you can shorten it this way